src/OpenTelemetry.Instrumentation.AspNet/TracerProviderBuilderExtensions.cs

```diff
@@ -42,7 +42,9 @@ public static class TracerProviderBuilderExtensions
             var aspnetOptions = new AspNetInstrumentationOptions();
             configureAspNetInstrumentationOptions?.Invoke(aspnetOptions);
 
-            builder.AddDiagnosticSourceInstrumentation((activitySource) => new AspNetInstrumentation(activitySource, aspnetOptions));
+            builder.AddInstrumentation(() => new AspNetInstrumentation(aspnetOptions));
+            builder.AddSource(typeof(AspNetInstrumentation).Assembly.GetName().Name);
```

Comment:
please move the sourcename to a constant. There is inconsistency now as this uses AspNetInstrumentation vs HttpInListener actually. (name is ultimately same, but lets do this consistently)

Reply:
I have updated the code to use ActivitySourceName from the listener implementation for all four instrumentation libraries